Source/Classes/PINRemoteImageManager.h

```diff
@@ -60,6 +60,14 @@ typedef NS_OPTIONS(NSUInteger, PINRemoteImageManagerDownloadOptions) {
     PINRemoteImageManagerDownloadOptionsIgnoreCache = 1 << 4,
     /** Skip download retry */
     PINRemoteImageManagerDownloadOptionsSkipRetry = 1 << 5,
+    /**
+     * Do not honor HTTP Cache-Control headers
+     * Currently PINRemoteImage will by default respect 'no-store', 'no-cache', 'max-age', 'Expires', and 'must-revalidate'. Set this flag to ignore those headers.
+     * TODO: Currently PINRemoteImage will re-download images that only must be re-validated. In the future this could be improved with revalidation behavior that stores ETag or Last-Modified values and only
```

Comment:
Minor / optional: "By default, PINRemoteImage will respect...".

Could also drop "Currently" in the next line, as presumably that will be updated if it becomes supported in the future. The next line also seems like it could use a line break to stay within a ~120 (or a bit more) column width — which isn't a hard rule for these frameworks, but up to ~150 works pretty well for Obj-C.

Great comment overall, thanks for including the IETF and Mozilla links!